data_steward/tools/generate_combined_dataset.sh

```diff
@@ -95,7 +95,7 @@ export UNIONED_DATASET_ID="${unioned_ehr_dataset}"
 export COMBINED_DATASET_ID="${combined_backup}"
 export BIGQUERY_DATASET_ID="${unioned_ehr_dataset}"
 
-bq mk --dataset --description "${version} combined raw version of  ${rdr_dataset} + ${unioned_ehr_dataset}" ${app_id}:${combined_backup}
+bq mk --dataset --description "${version} combined raw version of  ${rdr_dataset} + ${unioned_ehr_dataset}" --label "type:combined_raw" --label "release_tag:${dataset_release_tag}" --label "de_identified:false" ${app_id}:${combined_backup}
```

Comment:
This might be something to discuss in the engineering channel at large.  But I think this type could just be `backup` and the `type` on line 18 could just be `staging`.  That way we can easily get list all backups for a release_tag with a query like:
```
SELECT
 * 
FROM
 INFORMATION_SCHEMA.SCHEMATA_OPTIONS
 where option_name = 'labels' and 
option_value like '%("type", "backup")%'
-- instead of this
-- option_value like '%("type", "combined_raw")%' or option_value like '%("type", "backup")%'
-- I tested with this.
-- option_value like '%("release_tag", "dcampbell_dc1069_test")%'
```